lib/lotus/mailer/configuration.rb

```diff
@@ -183,10 +182,65 @@ def copy!(base)
         end
       end
 
+      # Specify a global delivery method
+      #
+      # @param method [Symbol] delivery method
+      # @param options [Hash] optional settings
+      #
+      # @return [Array] an array containing the delivery method and the optional settings as an Hash
+      #
+      # @since 0.1.0
+      #
+      # @example With Method only
+      # class DeliveryMethodSendmail
+      #   include Lotus::Mailer
+      #
+      #   Lotus::Mailer.configure do
```

Comment:
the Mailer configuration should be not in the mailer class, it should be called separately:

```
class DeliveryMethodSendMail
  include Lotus::Mailer
end

Lotus::Mailer.confgure do
  delivery_method: sendmail
end
```